ydb/_grpc/grpcwrapper/common_utils.py

```diff
@@ -156,13 +161,18 @@ async def start(self, driver: SupportedDriverType, stub, method):
             await self._start_sync_driver(driver, stub, method)
         self._connection_state = "started"
 
+    def close(self):
+        self.from_client_grpc.put_nowait(None)
```

Comment:
maybe it's better to use here something more specified instead of None? StopAsyncIteration object for example.

Reply:
good point, but StopAsyncIteration is bad decision, because it can handle both in sync and async iterator.
Done with special stop marker.